lib/utils/getRequesterForBrowser.ts

```diff
@@ -0,0 +1,17 @@
+import { supportsSendBeacon, supportsXMLHttpRequest } from "./featureDetection";
+
+import { requestWithSendBeacon, requestWithXMLHttpRequest } from "./request";
```

Comment:
why are these together in `./request`, and not in here directly?

Reply:
I think it seems okay that all kind of requests are in `request.ts` and the functions returning a request are in `getRequesterForXXX.ts`. What do you think?

Reply:
My point was to make sure that the bundle for browser doesn't get node code & vice versa

Reply:
Ah, yeah. I've checked the bundle output and it didn't happen.